knowledge-base/grid-state-filter-nullable-bool.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Programatically filter a nullable bool column in the Grid by null values
+description: Learn how to programatically filter nullable bool values
+type: how-to
+page_title: How to filter a Grid column bound to a nullable bool by the null values programatically
+slug: grid-kb-state-filter-nullable-bool
+tags: grid, blazor, column, datagrid, filter, state, nullable, bool
+res_type: kb
+ticketid: 1658561
+---
+
+## Environment
+
+<table>
+    <tbody>
+        <tr>
+            <td>Product</td>
+            <td>Grid for Blazor</td>
+        </tr>
+    </tbody>
+</table>
+
+## Description
+
+I want to programatically filter a Grid column that is bound to `bool?` by the `null` values. 
+
+
+This KB article also answers the following questions:
+- How to filter a Grid column bound to a `bool?` by the null values programatically
```

Comment:
This is the same sentence as the one above (the keywords and the message are the same). You can simply delete lines 28 and 29.

Reply:
I did this to search by statement and by question. I am unsure if that is correct since I am not very familiar with search engines, however, I can remove one or the other.